maintainers/maintainer-list.nix

```diff
@@ -2648,6 +2648,11 @@
     github = "lasandell";
     name = "Luke Sandell";
   };
+  lambda-11235 = {
```

Comment:
Please add yourself in a separate commit.

Reply:
How would I do that now that I've commited the package? The maintainer line needs to be there to build the package, which means I'd have to commit it before the package right? Any thoughts?

Reply:
Yeah, you can create it as a commit before. You'll need to redo the commits again.
One to add yourself and the other to add the package. You know how to use `git commit --amend` and `git rebase`?

Reply:
Ok, should be good now.

Reply:
No, please redo your commits so there are only 2 on this branch: `maintainers: add lambda-11235` and `redeclipse: init at 1.6.0`.

You can use `git rebase` for that.

Reply:
How is it now?

Reply:
Perfect